src/main/java/uk/gov/hmcts/reform/pip/data/management/service/publication/PublicationSearchService.java

```diff
@@ -14,13 +14,13 @@
 
 @Slf4j
 @Service
-public class ArtefactSearchService {
+public class PublicationSearchService {
     private final ArtefactRepository artefactRepository;
-    private final ArtefactService artefactService;
+    private final PublicationRetrievalService artefactService;
```

Comment:
rename artefactService to publicationRetrievalService

Reply:
changed to publicationRetrievalService